### DISCUSSION TITLE
YouTube thumbnails

### DISCUSSION THREAD

**Discussion by Sagar0-0** [gautamkrishnar/blog-post-workflow#135] [General]
Is there any way to get thumbnails from our YouTube videos? I tried using 'template' but still can't find my way to get the thumbnail and use it as a clickable link to the video(same as of title).

**Comment by Sagar0-0** [comment]
Can't we make it look something like this :

![githb](https://user-images.githubusercontent.com/85388413/166036227-e4bc3d69-3fe8-427d-a1af-8a420e022fb7.png)

**Comment by gautamkrishnar** [comment]
@Sagar0-0 please check the following urls for a similar example:
- https://github.com/Dexters-Hub/Dexters-Hub/blob/master/README.md#-latest-youtube-videos
- https://github.com/Dexters-Hub/Dexters-Hub/blob/master/.github/workflows/youtube-workflow.yml

You might need to update the template a little bit to achieve what you are looking for. It is just simple HTML.

**Comment by Sagar0-0** [comment]
Thanks a lot. It worked🔥🔥🔥
